### DISCUSSION TITLE
Documentation & Fine tuning

### DISCUSSION THREAD

**Discussion by duvi86** [amazon-science/chronos-forecasting#45] [Q&A] [answered]
Hello,

Great contribution! Where would be defined what all parameters represent and how to choose which one is the best for us? Is there a way to enter a multiple time series forecasting way? Typically, we have 1000 timeseries, somehow correlated or not. IS there an aggregate way to do this more effectively than 1 timeseries at a time?
How can we fine tune the model to our data?

Thanks!

**Comment by lostella** [comment, marked as answer]
@duvi86 The [docstring](https://github.com/amazon-science/chronos-forecasting/blob/2042779efa3aa2ad48bcefad34076cc57c0b0620/src/chronos/chronos.py#L390-L426) of the `predict` method contains a description of the parameters. You can leave the default ones to start, and only provide the time series and required `prediction_length`. A batch of multiple time series can be passed in in a list: note that the maximum allowed batch size depends on the amount of available memory (GPU or CPU) and prediction length and number of samples; this will typically be faster than predicting one series at a time. Note that each series will be predicted independently of the others (the model is univariate).

For fine tuning, please see #12.

**Comment by rasyousefi** [comment]
Can we finetune the model with our own dataset ?

**Reply by lostella** [reply]
For fine tuning, please see #12

**Comment by rasyousefi** [comment]
Can we add missing samples into the input time series provided to the model ? If yes, how can we do that ? 
Does the preprocessing tools looks for input time series at regular intervals ?

**Reply by lostella** [reply]
Yes, just use `torch.nan` in the series, wherever observations are missing. The model assumes that input time series have observations at regular intervals.

**Comment by abdulfatir** [comment]
We have released the training/fine-tuning code. Check out the examples here: https://github.com/amazon-science/chronos-forecasting/tree/main/scripts
